src/utils.ts

```diff
@@ -28,17 +44,51 @@ export function matchGlobs(filepath: string, globs: string[]) {
   return false
 }
 
-export function getNameFromFilePath(filePath: string): string {
+export function getNameFromFilePath(filePath: string, includedFrom: string | string[], allowFolderNames: boolean, namespaces: string[]): string {
```

Comment:
We can pass the entire options here

```suggestion
export function getNameFromFilePath(filePath: string, options: Options): string {
```

Reply:
Right! Completely oversaw that 👍